test/fuzz/Fuzz.t.sol

```diff
@@ -72,6 +73,7 @@ abstract contract Fuzz_Lending_Test is Base_Lending_Test, Fuzz_Test {
         pool = new LendingPoolExtension(users.riskManager, asset, treasury, address(factory), address(liquidator));
         srTranche = new TrancheExtension(address(pool), 0, "Senior", "SR");
         jrTranche = new TrancheExtension(address(pool), 0, "Junior", "JR");
+        trancheWrapper = new TrancheWrapper(asset, "SeniorWrapper", "SRW", address(srTranche));
```

Comment:
No need to add it to Fuzz.sol (-> will be deployed for every test in this repo).
Only declare it in _TrancheWrapper.sol